trymito.io/pages/spreadsheet-automation.tsx

```diff
@@ -0,0 +1,334 @@
+              <CTAButtons variant={'download'} align='left'/>
+            </div>
+            <div className={classNames(spreadsheetAutomationStyles.hero_video_container)}>
+              <video autoPlay loop disablePictureInPicture playsInline webkit-playsinline="true" muted>
+                  <source src="/formula_writing.mp4" />
+              </video>
+            </div>
+          </section>
+
+          <section className={textImageSplitStyles.text_image_section_container}>
+            <div className={pageStyles.subsection}>
+              <div className={classNames(textImageSplitStyles.functionality_media, 'display-desktop-only-inline-block')}>
+                <Image src={'/code_snippet.png'} alt='Mito generated code' width={500} height={250} layout='responsive'/>
+              </div>
+              <div className={textImageSplitStyles.functionality_text}>
+                <h1>
+                  The easiest way to <span className='text-highlight'>write Python code</span>
+                </h1>
+                <p>
+                  Every edit made in the Mito spreadsheet is automatically converted to Python code. 
+                </p>
+                <p>
+                  Edit your data just like you do in Excel and receive production ready Python code without searching Pandas documentation or Stack Overflow.
+                </p>
+              </div>
+              <div className={classNames(textImageSplitStyles.functionality_media, 'display-mobile-only-block')}>
+                <Image src={'/code_snippet.png'} alt='Mito generated code!' width={500} height={250} layout='responsive'/>
+              </div>
+            </div>
+            <div className={pageStyles.subsection}>
+              <div className={textImageSplitStyles.functionality_text}>
+                <h1>
+                  <span className='text-highlight'>Explore</span> your data visually
+                </h1>
+                <p> 
+                  There&apos;s no replacement for scrolling through your data. Enter fullscreen mode and build intuition about your analysis.
+                </p>
+                <p>
+                  Use Mito&apos;s Excel-like interface to view CSV/Excel files and dataframes. And further explore your data by graphing, viewing summary stats, and creating pivot tables.
+                </p>
+              </div>
+              <div className={classNames(textImageSplitStyles.functionality_media, textImageSplitStyles.functionality_media_supress_bottom_margin)}>
+                <Image src={'/explore_data_visually.png'} alt='Explore data with Mito' width={500} height={250} layout='responsive'/>
+              </div>
+            </div>
+          </section>
+
+          <section className={pageStyles.background_card}>
+
+            <div className={classNames(pageStyles.subsection, pageStyles.subsection_column, 'center')}>
+              <h2>
+                All of the components you need, built to generate Python code
```

Comment:
```suggestion
                All of the functionality you expect, built to generate Python code
```

Reply:
<img width="800" alt="Screen Shot 2023-07-05 at 2 11 32 PM" src="https://github.com/mito-ds/monorepo/assets/21002431/318453fd-0791-413f-b247-f3e9aeacee84">


every image is just a bit squished. Can we fix that?

Reply:
<img width="662" alt="Screen Shot 2023-07-05 at 2 12 18 PM" src="https://github.com/mito-ds/monorepo/assets/21002431/ba868b2c-c889-4980-918e-cdd1bee92c68">

my eyes. wanna take another pass?